kafka_utils/kafka_consumer_manager/commands/offset_get.py

```diff
@@ -140,7 +181,8 @@ def get_offsets(cls, client, group, topics_dict, storage):
     @classmethod
     def print_output(cls, consumer_offsets_metadata, watermark_filter):
         for topic, metadata_tuples in consumer_offsets_metadata.iteritems():
-            print ("Topic Name: {topic}".format(topic=topic))
+            diff_sum = sum([t.highmark - t.current for t in metadata_tuples])
+            print ("Topic Name: {topic}  Total difference: {diff}".format(topic=topic, diff=diff_sum))
```

Comment:
I'm not sure about total difference here, can we name it something like offset difference?